homeassistant/components/tradfri/base_class.py

```diff
@@ -128,3 +137,8 @@ def device_info(self) -> DeviceInfo:
             sw_version=info.firmware_version,
             via_device=(DOMAIN, self._gateway_id),
         )
+
+    def _refresh(self, device: Device, write_ha: bool = True) -> None:
+        """Refresh the device data."""
+        self._attr_available = device.reachable
```

Comment:
Maybe add a comment here why we can't put this in the base class.

Reply:
Done. If you want another text, feel free to change it.